transaction/core/src/lib.rs

```diff
@@ -33,10 +41,8 @@ pub mod encrypted_fog_hint;
 pub mod fog_hint;
 pub mod membership_proofs;
 pub mod mint;
-pub mod onetime_keys;
 pub mod range_proofs;
-pub mod ring_signature;
-pub mod signer;
+pub mod ring_ct;
```

Comment:
I wanted to put the `SignatureRctBulletproofs` object in a module called `ring_ct` rather than `ring_signature`, and in code, try not to call that thing a "Ring Signature".

I think it's more correct to say that a `RingMLSAG` is a ring signature, and the Rct bulletproofs construction is an application of "Ring Confidential Transactions" (RingCT). So one goal I have here is to try to separate and reduce confusion between these two concepts.

Reply:
💯